src/components/Book.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+function Book({ book }) {
+  const { title, author } = book;
+  return (
+    <li className="flex w-full md:w-1/2 justify-between items-center gap-4 list-none my-4">
+      <p>
+        <span>{title}</span> by <span>{author}</span>
+      </p>
+      <button className="border-2 border-black px-2 py-1">delete</button>
+    </li>
+  );
+}
+Book.propTypes = {
+  book: PropTypes.object,
```

Comment:
This proptypes is not very strict and efficient... A `book` is an object, however, the properties are not specified which means any property can be passed into it

Reply:
Done,
Make use of Prototypes.shape function to specify the value expected on each key